tests/test_sftp.py

```diff
@@ -277,7 +278,7 @@ def test_listdir_in_locale(self, sftp):
         sftp.open(sftp.FOLDER + "/canard.txt", "w").close()
         try:
             folder_contents = sftp.listdir(sftp.FOLDER)
-            self.assertEqual(["canard.txt"], folder_contents)
+            TestCase().assertEqual(["canard.txt"], folder_contents)
```

Comment:
What is your reasoning behind this replacement? This seems like it will unnecessarily create a new, empty `TestCase` instance for each assert, instead of just using the one that already exists in the form of `self`.

Reply:
Hmm? I don't remember changing this myself. I'll check when I come home tomorrow night.

Thank you.